crates/ra_proc_macro/src/rpc.rs

```diff
@@ -34,15 +34,15 @@ pub struct ListMacrosResult {
 pub struct ExpansionTask {
     /// Argument of macro call.
     ///
-    /// In custom derive that would be a struct or enum; in attribute-like macro - underlying
+    /// In custom derive this will be a struct or enum; in attribute-like macro - underlying
     /// item; in function-like macro - the macro body.
     #[serde(with = "SubtreeDef")]
     pub macro_body: Subtree,
 
-    /// Names of macros to expand.
+    /// Names of macros to expand. // TODO: are they comma-separated?
     ///
     /// In custom derive those are names of derived traits (`Serialize`, `Getters`, etc.). In
```

Comment:
@Veetaha And these lines too, We only parse a single name now.

Reply:
Oh, yeah, forgot to stage change)